### PR TITLE
added new sound class to play sounds. Also sound when removing a block

### PR DESCRIPTION
**Pull request by xieWright**
Added a Sound class to deal with sounds as well as a sound list file to deal with compiling and organizing sounds. Currently added a sound when removing blocks.

### CODE REVIEW

**Review by criticalglitch — CHANGES_REQUESTED** [review]
It doesn't seem the sound plays multiple times if you delete more than one block. It should play the sound on every block break.

**Review by criticalglitch — APPROVED** [review]
I think this looks good now. Every broken block played a sound, and when doing it rapidly, did not crash the game.

File: assets/sound/rock_hit.wav
Comment (w1158mxs) [inline assets/sound/rock_hit.wav:1]: I Suggest getting all sounds in one folder called "sounds"

**Review by w1158mxs — APPROVED** [review]
Good Job, Suggest getting all wav files in one folder.

**Review by Lurk390 — APPROVED** [review]
The changes look good. My only suggestions (and these would probably be new issues) is to either decrease the base volume of the effect or add a sound slider in the pause menu.

**Review by bakursait2 — APPROVED** [review]
The sound effect is good addition to the game.
As suggestion (maybe a new issue) is to make a sound for removing a block.

Overall it is good addition to the project.

File: assets/sound/rock_hit.wav
Comment (xieWright) [reply assets/sound/rock_hit.wav:1]: They are in the assets under the Sound folder

**Pull request merged**